flink-examples/flink-examples-batch/src/main/java/org/apache/flink/examples/java/clustering/KMeans.java

```diff
@@ -105,119 +110,127 @@ public static void main(String[] args) throws Exception {
 			.groupBy(0).reduce(new CentroidAccumulator())
 			// compute new centroids from point counts and coordinate sums
 			.map(new CentroidAverager());
-		
+
 		// feed new centroids back into next iteration
 		DataSet<Centroid> finalCentroids = loop.closeWith(newCentroids);
-		
+
 		DataSet<Tuple2<Integer, Point>> clusteredPoints = points
-				// assign points to final clusters
-				.map(new SelectNearestCenter()).withBroadcastSet(finalCentroids, "centroids");
-		
+			// assign points to final clusters
+			.map(new SelectNearestCenter()).withBroadcastSet(finalCentroids, "centroids");
+
 		// emit result
-		if (fileOutput) {
-			clusteredPoints.writeAsCsv(outputPath, "\n", " ");
+		if (params.has("output")) {
+			clusteredPoints.writeAsCsv(params.get("output"), "\n", " ");
 
 			// since file sinks are lazy, we trigger the execution explicitly
 			env.execute("KMeans Example");
-		}
-		else {
+		} else {
 			clusteredPoints.print();
```

Comment:
Add a message saying something like "Printing result to std-out. Use --output to specify output path."